internal/node/node_repositories.bzl

```diff
@@ -188,7 +191,11 @@ def _prepare_node(repository_ctx):
     Args:
       repository_ctx: The repository rule context
     """
-    is_windows = os_name(repository_ctx).find("windows") != -1
+    is_windows_os = os_name(repository_ctx).find("windows") != -1
+
+    # TODO: Maybe we want to encode the OS as a specific attribute rather than do it based on naming?
+    is_windows_repository = repository_ctx.attr.name.find("windows") != -1
+    is_windows = is_windows_os or is_windows_repository
```

Comment:
this doesn't seem right. what matters is if the repository is windows when preparing it not if the host os is right? I'll do some testing on Windows tomorrow.

nit: also `is_windows_repository = repository_ctx.attr.name.find("_windows_") != -1` is a bit safer.

Reply:
Yeah, now thinking about it I think you are right. We care about which os the repository is for, so cross compilation on windows might indeed be broken with this. We only care about the host os when we run `repository_ctx.execute` as well as for the alias repository `nodejs`. So maybe we should change this to `(is_windows_os and repository_ctx.attr.name == "nodejs") or ...`, or as my comment says above we could also start passing in an OS attribute to be a bit more generic. WDYT?